frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -436,9 +436,9 @@ export const ADMIN_SETTINGS_SECTIONS = {
   },
   "embedding-in-other-applications/full-app": {
     // We need to do this because EE plugins would load after this file in unit tests
-    component: ({ updateSettings }) => (
+    component: ({ updateSetting }) => (
```

Comment:
I passed the wrong prop, so this wasn't working. I added a test to ensure correct prop above.